packages/web/components/bridge/immersive/amount-and-review-screen.tsx

```diff
@@ -39,6 +39,12 @@ export const AmountAndReviewScreen = observer(
     const [cryptoAmount, setCryptoAmount] = useState<string>("0");
     const [fiatAmount, setFiatAmount] = useState<string>("0");
 
+    // if the chains change, reset the amounts
+    useEffect(() => {
```

Comment:
I prefer predictable changes when it comes to state management. I think it's a better idea to handle this change in the `onSelectChain` function on the amount screen